src/components/DateInput.jsx

```diff
@@ -117,7 +117,7 @@ export default class DateInput extends React.Component {
           value={value}
           onChange={this.onChange}
           onKeyDown={this.onKeyDown}
-          onFocus={onFocus}
+          //onFocus={onFocus}
```

Comment:
So this is unlikely to be the solution given that this likely breaks keyboard navigation.